app/models/activity.rb

```diff
@@ -0,0 +1,22 @@
+class Activity
+  include ActiveModel::Serialization
+
+  TYPE = 'Prison Activities'.freeze
+
+  attr_reader :id, :start_time, :reason, :type, :agency_id
+
+  def build_from_nomis(activity)
+    @id = activity['eventId']
+    # Nomis startTime does not include a zone so we shouldn't use our local zone with Time.zone.parse
```

Comment:
@willfish can you clarify in what timezone Nomis times are provided? If it is local time (BST / GMT) **and the server runs in the same timezone**, then Time.zone.parse should be fine. Otherwise, if Nomis is in UTC (cf. `app/services/court_hearings/create_in_nomis.rb`) then I think we should be forcing the timezone in the parse.

E.g. consider the difference:

```
Time.zone.parse('19:00')
=> Mon, 27 Apr 2020 19:00:00 BST +01:00
Time.zone.parse('19:00 UTC')
=> Mon, 27 Apr 2020 20:00:00 BST +01:00
```